src/course-unit/data/slice.js

```diff
@@ -9,6 +9,7 @@ const slice = createSlice({
     savingStatus: '',
     isQueryPending: false,
     isEditTitleFormOpen: false,
+    canEdit: true,
```

Comment:
[optional] canEdit: false

Reply:
Now for testing, we always want to display the copy-paste functionality